### DISCUSSION TITLE
目前的动画非常不友好，小程序已经升级，taro是否跟进

### DISCUSSION THREAD

**Discussion by andyli09** [NervJS/taro#12928] [Q & A] [answered]
### 这个特性解决了什么问题？
现在taro做动画使用的Taro.createAnimation 这个api非常难用，经常卡顿，而且一个页面中多个动画会相互影响。所以，微信小程序为了解决此问题，已经优化升级了，升级说明可以见https://developers.weixin.qq.com/miniprogram/dev/framework/view/animation.html 咱们taro是否会跟进升级。

### 这个 API 长什么样？
期望taro能尽快升级跟进，目前的这个.createAnimation 太难用了

**Comment by TheKonka** [comment]
直接把小程序中的`this`换成`Taro.getCurrentInstance().page` 试一下

**Comment by andyli09** [comment]
> Taro.getCurrentInstance().page

这个getCurrentInstance和animation有什么关系，里面没有动画属性呢，能把taro如何使用小程序最新的动画方法说的仔细一些吗

**Comment by TheKonka** [comment, marked as answer]
> > Taro.getCurrentInstance().page
> 
> 这个getCurrentInstance和animation有什么关系，里面没有动画属性呢，能把taro如何使用小程序最新的动画方法说的仔细一些吗

`Taro.getCurrentInstance().page` 就是返回小程序的 `page` 对象 ，`Taro.getCurrentInstance().page?.animate` 就等于 `this.animate`

![image](https://user-images.githubusercontent.com/22173084/205472549-67517f72-aefe-481b-baf4-dd68680d79a1.png)

可以参考这个人的写法
https://github.com/NervJS/taro/issues/11700
